### ISSUE TITLE
Categorisation data to be stored for API

### ISSUE THREAD

**Issue by Greatlemer** [HackBrexit/MinistersUnderTheInfluence#93]
### Description
We will need to store information of the category each organisation meeting with the government belongs to

#### Comments, Questions and Considerations
Categorisation of the organisations will be a manual process so may need an interface to do this or an API to upload a batch of data into.

**Comment by charlottebrf** [comment]
@Greatlemer Can I close this?

**Comment by aspiers** [comment]
@charlottebrf I don't think so, since we have nowhere to store this categorisation in the current data model.

@Greatlemer by "category" here, do you mean industry sectors like finance, energy, health etc.?  If so, I'm wondering whether a set of categories or even a hierarchy might be too inflexible.  For example a health insurance company could be categorised under both finance and health, and Facebook / Google could be categorised under both advertising and IT (and let's face it, IT is a very vague "sector" anyway since every sector uses IT).

So I'm thinking it might make more sense to adopt a "tags" approach, i.e. with a many-to-many relationship between tags and organisations.

I think this should be a stretch goal for the showcase.

**Comment by charlottebrf** [comment]
I like the idea of tags @aspiers it's something we've discussed before. One challenge we've previously discussed is the subjectivity of creating tag classes/ how we can be consistent. There will always be a discrepancy in how people search for tags e.g. I doubt I would search IT but maybe just technology as a keyword, and every user will bring their own idiosyncrasies.  I'm seeing Alex from Unlock Democracy on Thurs so will discuss with her also.

**Comment by aspiers** [comment]
I don't think the subjectivity matters too much as long as all tags are easily discoverable (e.g. via a list and auto-completion).  Hmm, although if it's not a many-to-one relationship between organisations and tags/categories/sectors/whatever you want to call them, but rather many-to-many, then that breaks the idea of drill-down bubbles with categories at the top as per #105.  So maybe we should stick with many-to-one.

**Comment by JohnSmall** [comment]
Do we have some examples I can work with? It's pretty easy to add tags to the organisations, there are gems for this. But examples of how you want to CRUD the tags are essential. Via the API, via a UX?

**Comment by aspiers** [comment]
@JohnSmall Since it's important for us to be able to display one bubble per industry sector (without double-counting organisations in more than one bubble), ideally in time for the showcase, I'd suggest that for now we start with non-overlapping categories.  Here's one list we might use:

- https://en.wikipedia.org/wiki/United_Kingdom_Standard_Industrial_Classification_of_Economic_Activities#Current_2007_SIC_codes

Later if we want to be able to tag organisations into multiple groups (i.e. a many-to-many relationship) then we can add tags as a secondary mechanism for grouping, but I think we should start with non-overlapping categories (i.e. a one-to-many relationship) for the reason stated above.  (N.B. I kind of stole this categories vs. tags distinction from the way Wordpress facilitates grouping of blog posts - it works well there, so it should work well for us too I think.)

Regarding categories, we simply need to add a new model and corresponding resource to the Rails app, with a one-to-many relationship with organisations.

**Comment by aspiers** [comment]
This came up in discussion with Alex Runswick at the showcase. She did emphasise the difficulty and danger of placing organisations into one of a discrete number of sector "buckets".
